cg/cli/upload/vogue.py

```diff
@@ -234,7 +233,7 @@ def _get_samples(store: Store, case_name: str) -> str:
         sample_names(str): ACC12345,ACC45679
     """
 
-    link_objs = get_links(store, case_name)
+    link_objs = store.family(case_name).links
```

Comment:
TYPEHINT!

Reply:
Only this once. I wont refactor this entire file with typehints. This can get REALLY huge if i did